recipes/tmuxp/conda-forge.yml

```
@@ -0,0 +1,3 @@
+noarch_platforms:
+  - linux_64
+  - win_64
```

My guess is that you want it on macOS but not Windows, right?

```suggestion
  - osx_64
```